ACHNBrowserUI/ACHNBrowserUI/fr.lproj/Localizable.strings

```diff
@@ -489,7 +489,7 @@ You can cancel anytime with your iTunes account settings. Any unused portion of
 "On rotten turnips" = "Sur les navets pourris";
 "Shaking trees" = "En secouant les arbres";
 "On white flowers" = "Sur les fleurs blanches";
-"Flying near trash (boots, tires, cans)" = "Vole près des ordures (bottes, pneus, canettes)";
+"Flying near trash (boots, tires, cans) or rotten turnips" = "Vole près des ordures (bottes, pneus, canettes) ou prêt des navets pourris";
```

Comment:
I'm not sure about this one as you changed the key, I think it won't work as expected. But the translation is good!
```suggestion
"Flying near trash (boots, tires, cans)" = "Vole près des ordures (bottes, pneus, canettes) ou prêt des navets pourris";
```

Reply:
The data changed and this fixes a broken translation